tools/gce-quota-sync/gce-quota-sync.py

```diff
@@ -39,6 +40,9 @@
 _METRIC_KIND = monitoring_v3.enums.MetricDescriptor.MetricKind.GAUGE
 _METRIC_TYPE = 'custom.googleapis.com/quota/gce'
 
+# set project_id to avoid auth errors. This should be moved into main.
+os.environ["GOOGLE_CLOUD_PROJECT"] = project_id
```

Comment:
Where is the variable `project_id` defined? I cannot find it in this file.

Reply:
The project_id is not since we can expect the value to change based on who's running this. It should be the same value as the as the project_id provided in the _add_series function. In theory, this variable can be set in the main function but it probably makes sense to move it to the _add_series function since it's been predefined there.